src/python/pants/build_graph/build_file_parser.py

```diff
@@ -61,127 +61,3 @@ def root_dir(self):
     def registered_aliases(self):
         """Returns a copy of the registered build file aliases this build file parser uses."""
         return self._build_configuration.registered_aliases()
-
```

Comment:
Note: this is all dead code. It was dead code before this PR, too.

Reply:
Could you please break this out into a dedicated PR? 

This change looks incredible, although the size of the diff makes it a little overwhelming to review. It’d be hugely helpful to break out any possible prework PRs like this so that this PR is smaller.